config/zones/CA-ON.yaml

```diff
@@ -87,33 +87,26 @@ emissionFactors:
       - datetime: '2021-01-01'
         source: Electricity Maps, 2021 average
         value: 57.47323263502428
+    gas:
+      datetime: '2015-01-01'
+      source: Mallia, E., Lewis, G. "Life cycle greenhouse gas emissions of electricity
+        generation in the province of Ontario, Canada." (2013)
+      value: 497 
```

Comment:
Gas has combined lifecycle and direct emissions. This value is for total (lifecycle + direct) but has been put in the `lifecycle` section. Should this be in `direct` section instead? Or somewhere else altogether? @pierresegonne

Reply:
The reported figure is computed using life-cycle assessment so it's correct to put it in lifecyle. For direct, we could go through the paper, to see if they provide a breakdown of the emission factor, or else remove the upstream emissions from IPCC (120g CO2eq/kWh)